src/main.py

```diff
@@ -169,6 +169,14 @@ def pdf_uri(triple):
     filename = "elife-%s-v%s.pdf" % (utils.pad_msid(msid), version) # ll: elife-09560-v1.pdf
     return cdnlink(msid, filename)
 
+def xml_uri(triple):
+    """predict an article's xml url."""
+    content_type, msid, version = triple
+    if content_type and any(lmap(lambda type: type in ['Correction', 'Retraction'], content_type)):
```

Comment:
this is true for PDF (where this comes from) but I think the XML is always there for any article. I'll check.

Reply:
Ok thank you.